frontend/src/pages/EventCreationPage/components/EventCreationForm/index.tsx

```diff
@@ -29,9 +29,29 @@ interface InitialValuesType {
   hosts: OfficerNameData[];
 }
 
+let findNearestHalfHour = (date: Date) => {
+  let startMinutes = getMinutes(date);
+  if (startMinutes === 0) {
+    startMinutes = 0;
+  }
+  else if (startMinutes != 0 && startMinutes <= 30) {
+    date = add(date, { minutes: (30 - startMinutes) });
```

Comment:
This is fine, but since you're just setting it to 30 minutes anyway, this function is probably even easier to use: https://date-fns.org/v2.21.1/docs/setMinutes